turbopack/crates/turbo-tasks-fs/src/lib.rs

```diff
@@ -72,6 +72,80 @@ use crate::{
     rope::{Rope, RopeReader},
 };
 
+pub const MAX_FILE_NAME_LENGTH_UNIX: usize = 255;
```

Comment:
Is this going to generate compiler warnings on windows about dead code?

If so, I'd just inline the constant inside of `validate_path_length_inner` alongside `MAX_PATH_LENGTH`